expression/generator/other_vec.go

```diff
@@ -128,23 +131,112 @@ func (b *{{.SigName}}) vecEvalInt(input *chunk.Chunk, result *chunk.Column) erro
 		r64s[i] = 0
 	}
 	hasNull := make([]bool, n)
+	for i := 0; i < n; i++ {
+		if buf0.IsNull(i) {
+			hasNull[i] = true
+		}
+	}
```

Comment:
Why do we need consider `buf0` here?

Reply:
Seems it's redundant here. I will remove it.